exporters/otlp/internal/transform/span_test.go

```diff
@@ -93,7 +93,7 @@ func TestSpanEvent(t *testing.T) {
 	if !assert.Len(t, got, 2) {
 		return
 	}
-	uNow := uint64(now.Nanosecond())
+	uNow := uint64(now.UnixNano())
```

Comment:
Can you use a fixed time in the test so that the test doesn't use the same logic as the code under test for validation?